packages/ai/README.md

```diff
@@ -1,21 +1,233 @@
 # @rsdoctor/mcp-server
 
-## Usage with Claude Desktop or other MCP clients
+## Introduction
 
-Add to your claude_desktop_config.json or .cursor/mcp.json:
+@rsdoctor/mcp-server is an MCP Server tool designed to help users more conveniently utilize Rsdoctor's build data. It works with Rsdoctor's local build analysis data and helps you quickly obtain build analysis results through a Q&A interface.
+
+## Features
+
+Currently supports four main types of analysis capabilities:
+
+- **Build Artifact Analysis**: Analyzes the size and composition of build artifacts
+- **Dependency Analysis**: Analyzes project dependencies and duplicate dependency issues
+- **Bundle Optimization Suggestions**: Provides suggestions for bundle size optimization and code splitting
+- **Build Performance Suggestions**: Analyzes build time and provides build performance optimization suggestions
+
+## Examples
+
+### 1. Bundle Optimization Analysis
+
+By asking "Please help me to optimize the bundle or artifacts", the tool will analyze the build artifacts and provide optimization suggestions.
+
+Example video:
+
+https://github.com/user-attachments/assets/c73e55a9-445d-481d-8dbd-cf99bfca680f
+
+### 2. Dependency Analysis
+
+By asking "Please investigate the referrer dependency of node_modules/dayjs/index.js", the tool will analyze the dependency relationships of the specified module.
+
+Example video:
+
+https://github.com/user-attachments/assets/312cf5ce-19bd-49e7-87bb-aab1fc8a6a43
+
+### 3. Build Performance Analysis
+
+By asking "Please help me find files or loaders with high compilation time and provide optimization suggestions", the tool will analyze build time and provide optimization suggestions.
+
+Example video:
+
+https://github.com/user-attachments/assets/cc0f5441-4950-420c-bbad-635e21e87492
+
+## Configuration
+
+### Parameters
+
+#### compiler
+
+- Configure the name of the linked bundler (optional)
+
+```linux
+npx @rsdoctor/mcp-server@latest --compiler web
+```
+
+#### port
+
+- Configure the port (optional)
+
+```linux
+npx -y @rsdoctor/mcp-server@latest --port 1000
+```
+
+Note: By default, the Rsdoctor local server starts on a random port. Therefore, if you want to use the port parameter, you need to configure the port in the plugin:
+
+```js
+export default {
+  // ...
+  tools: {
+    bundlerChain: (chain) => {
+      if (process.env.RSDOCTOR) {
+        chain.plugin('Rsdoctor').use(RsdoctorRspackPlugin, [
+          {
+            port: 9988,
+          },
+        ]);
+      }
+    },
+  },
+};
```

Comment:
We should avoid using `tools.bundlerChain` as an example here, as users may be using Rspack CLI or other tools and frameworks.

```suggestion
new RsdoctorRspackPlugin({
  port: 9988,
});
```